test_runner/regress/test_subscriber_restart.py

```diff
@@ -0,0 +1,53 @@
+import threading
+
+from fixtures.neon_fixtures import NeonEnv
+from fixtures.utils import wait_until
+
+
+def test_subscriber_restart(neon_simple_env: NeonEnv):
```

Comment:
Can you write a docstring on this function that explains the purpose of the test -- as written, it's not really obvious that this is about logical replication unless the reader happens to know that "subscriber" means logical replication subscriber.